lib/storage/sqlite/campaign-info-storage.js

```diff
@@ -53,4 +53,22 @@ class CampaignInfoStorage {
   }
 }
 
+/**
+ * Fetch a CampaignInfoByCandidate object by id
+ *
+ * @param {Number} id candidate_id
+ * @param {campaignInfoProcessor} cb
+ */
+CampaignInfoStorage.prototype.fetchByCampaign = function (id, cb) {
+  db.open(this.dbpath)
+    .then((stmt) => stmt.all('SELECT * FROM campaign_info WHERE candidate_id = ?', id))
```

Comment:
> You can see an example of building that object in the candidate storage method I've written but not merged yet).

Yeah, I had a look at that.

> Probably the simplest thing to do here is just change the endpoint to /candidates/:id/campaigns/info, since the work is done for literally exactly that.

👍